assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -1,22 +1,20 @@
 .component-stripe-pop-up-button {
-  background-color: gu-colour(neutral-1);
+  box-sizing: border-box;
```

Comment:
I think `border-box` was a consequence of the button having a border. Now that there's no border it may not be needed.